src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -505,6 +505,7 @@ public static bool RunningOnPosix
         public OpenFlags PosixOpenFlags = SafePosixOpenFlags;
         public Win32NativeFileAttributes WinOpenFlags = SafeWin32OpenFlags;
         public DateTime? NonSafeTransactionExpiration { get; set; }
+        public int MaxConcurrentFlushes { get; set; }
```

Comment:
This shouldn't be an instance method, it should be a static method that has a reasonable default, and is wired in the ServerStore (only).